static/js/ranking/ranking_page.tsx

```diff
@@ -41,6 +44,27 @@ class Page extends React.Component<RankingPagePropType, RankingPageStateType> {
       data: undefined,
     };
   }
+
+  private renderToggle(): JSX.Element {
+    const svData = this.state.data;
+    if (svData.rankAll) return;
+    if (svData.rankBottom1000) {
+      // show link to top 100
```

Comment:
should this be bottom 1000?

Reply:
It should be top - we show the link to the opposite based on what's shown. In other words:

if (showing bottom 100)
-- show link to the top 100

Reply:
that makes sense!